### PR TITLE
TRIVIAL: Add PR template with checklist

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/214/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/256/ : SUCCESS in 1m 20s

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+_Description of changes (if multi-commit, short global summary & context;
+if single-commit, feel free to leave empty)._
+
+---
+
+# PR checklist
+
+- [ ] Change is described in CHANGELOG.md file.
```
Comment (martinmosko) [inline .github/pull_request_template.md]: CHANGELOG.md could be link to the file? :)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/217/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/260/ : SUCCESS in 48s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/64/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/262/ : SUCCESS in 43s

**Pull request merged**